src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/TestConfigureCallGen.generated.txt

```diff
@@ -1,91 +1,124 @@
 // <auto-generated/>
 #nullable enable
 
-using System.Linq;
-
 internal static class GeneratedConfigurationBinder
 {
-    public static global::Microsoft.Extensions.DependencyInjection.IServiceCollection Configure<T>(this global::Microsoft.Extensions.DependencyInjection.IServiceCollection services, global::Microsoft.Extensions.Configuration.IConfiguration configuration)
+    public static global::Microsoft.Extensions.DependencyInjection.IServiceCollection Configure<T>(this global::Microsoft.Extensions.DependencyInjection.IServiceCollection services, global::Microsoft.Extens
+ions.Configuration.IConfiguration configuration)
```

Comment:
Does this compile?

Reply:
My mistake. Doesn't seem straightforward to assert that the resulting compilation has no diagnostics given these test utils. If that's the case I can create an issue to extend the API to support that.